server/explore.js

```diff
@@ -33,11 +44,30 @@ export const updateUserExploreNumber = () => {};
 
 /*
 Function to get called when populating the explore feed view
-QUESTION: Where are we getting initial data from, facebook ?!
-
 */
-export const populateExploreFeed = () => {};
+export const populateExploreFeed = (user, conditions) => {
+  // show most recent articles first
+  conditions.sort = { _id: -1 };
```

Comment:
I figured/think I remember that we want to show more recently added articles first in the explore feed, but we can change this if we want to sort by something else.